### DISCUSSION TITLE
Easier opting out of deleting cookies and site data per website

### DISCUSSION THREAD

**Discussion by cryeprecision** [zen-browser/desktop#3536] [Ideas]
Add an option to opt out of deleting cookies and site data for a website within two or three clicks.

I'd like to enable the `Delete cookies and site data when Zen Browser is closed` option and then opt out of this for individual websites. As far as I'm aware, this currently requires going to the settings and manually adding an exception for the website in the `Manage Exceptions...` menu which this is currently preventing me from using this feature (I'm lazy).

The Brave browser has this feature and also exposes a number of other options through the menu in the URL bar, like whether to enforce HTTPS and what cookies to block. Having quick access to such options would be a huge quality of life improvement.

I'd imagine this to look somewhat like this
![image](https://github.com/user-attachments/assets/44973bd3-4f2f-420a-a6e0-6b293209b31b)

**Comment by Guchdorg** [comment]
In the meantime you can use the Firefox extension [Cookie Autodelete](https://addons.mozilla.org/en-US/firefox/addon/cookie-autodelete/?utm_source=addons.mozilla.org&utm_medium=referral&utm_content=search).
